components/Form/MultiSelectInput/index.jsx

```diff
@@ -91,41 +91,28 @@ const propTypes = {
 
 };
 
-const defaultProps = {
-    searchable: true,
-    clearable: true,
-    disabled: false,
-    loading: false,
-    placeholder: 'Select...',
-    keyExtractor: (item) => item.id,
-    valueExtractor: (item) => item.name,
-    options: [],
-    onChange: noop,
-    optionsDirection: 'down',
-};
-
 const MultiSelect = ({
     name,
     className: _className,
     controlClassName,
     optionsWrapperClassName,
     selectOptionClassName,
-    loading,
-    disabled,
-    clearable,
-    searchable,
-    placeholder,
-    keyExtractor,
-    valueExtractor,
+    loading = false,
+    disabled = false,
+    clearable = true,
+    searchable = true,
```

Comment:
`clearable` and `searchable` aren't used anywhere.

Reply:
removed clearable and support searchable